### ISSUE TITLE
High contrast theme

### ISSUE THREAD

**Issue by timotheeguerin** [Azure/BatchExplorer#1762]
Support for high contrast theme:
* Should have the right colors
* Should switch to that theme automatically when OS switch to high contrast mode
